frame/auction.ts

```diff
@@ -35,29 +37,43 @@ import { FetchJsonStatus, tryFetchJson } from "./fetch";
  *
  * @param hostname The hostname of the page where the FLEDGE Shim API is
  * running.
+ * @param extraScript Additional JavaScript code to run in the Web Worker before
+ * anything else. Used only in tests to stub out `console.warn`.
```

Comment:
nit: "Used only in tests to stub out `console.warn`." -> "Used only in tests."

If later on we want to make additional test only changes we will probably forget to update this comment